man/systemd.network.xml

```diff
@@ -2012,6 +2012,14 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UseIPv6OnlyPreferred=</varname></term>
```

Comment:
Even if the option name contains `preferred`, the setting should be `IPv6OnlyMode=` or `AllowIPv6OnlyMode=`.